app/controllers/providers/proceedings_types_controller.rb

```diff
@@ -4,23 +4,26 @@ class ProceedingsTypesController < ProviderBaseController
 
     # GET /provider/applications/:legal_aid_application_id/proceedings_types
     def index
+      @multiple_proceedings = Setting.allow_multiple_proceedings?
       proceeding_types
     end
 
     # POST /provider/applications/:legal_aid_application_id/proceedings_types
     def create
       return continue_or_draft if draft_selected?
 
-      if legal_aid_application.proceeding_types.present?
+      if run_transaction
         go_forward
       else
         legal_aid_application.errors.add(:'proceeding-search-input', t('.search_and_select'))
+        @multiple_proceedings = Setting.allow_multiple_proceedings?
```

Comment:
you could maybe stick lines 7 and 11 in their own private method and call them from there to avoid duplication. Or maybe not have these in the controller at all and just have `Setting.allow_multiple_proceedings?` in the relevant places in the template so they are easier to locate and remove when this feature flag isn't required anymore maybe.

Reply:
I stuck with a private method. I think using Setting.allow_multiple_proceedings? in the view is less visible and introduces more logic in the view than neccessary. Just preference tbf.